sigridci/sigridci/junit_format_report.py

```diff
@@ -43,13 +44,11 @@ def generateXML(self, feedback, options):
         return dom.toprettyxml(indent="    ")
 
     def getFailures(self, feedback, options):
-        if self.meetsObjectives(feedback, options):
-            return []
-
+        status = Objective.determineStatus(feedback, options)
         failures = []
 
-        for metric in self.REFACTORING_CANDIDATE_METRICS:
-            if not self.meetsObjectives(feedback, options):
+        if status == ObjectiveStatus.WORSENED:
```

Comment:
As discussed this is the desirable situation (at least for now), as we don't want to have a full failure because of the refactoring candidates findings when we actually see Sigrid CI as succeeded. So this is the correct solution now.